storage/fuzz/fuzz_targets/journal_operations.rs

```diff
@@ -10,20 +10,41 @@ use commonware_storage::journal::fixed::{
 use futures::{pin_mut, StreamExt};
 use libfuzzer_sys::fuzz_target;
 
+const MAX_REPLAY_BUF: usize = 2048;
+
+fn bounded_non_zero(u: &mut Unstructured<'_>) -> Result<usize> {
+    let v = u.int_in_range(1..=MAX_REPLAY_BUF)?;
+    Ok(v)
+}
+
 #[derive(Arbitrary, Debug, Clone)]
 enum JournalOperation {
-    Append { value: u64 },
-    Read { pos: u64 },
+    Append {
+        value: u64,
+    },
+    Read {
+        pos: u64,
+    },
     Size,
     Sync,
-    Rewind { size: u64 },
+    Rewind {
+        size: u64,
+    },
     OldestRetainedPos,
-    Prune { min_pos: u64 },
-    Replay,
+    Prune {
+        min_pos: u64,
+    },
+    Replay {
+        #[arbitrary(with = bounded_non_zero)]
```

Comment:
I guess I expected you'd need something like this before L40.  Or does this apply to all values below it?